c2rust-analyze/src/main.rs

```diff
@@ -180,11 +315,36 @@ fn run(tcx: TyCtxt) {
     }
 }
 
-fn assign_pointer_ids<'tcx>(acx: &mut AnalysisCtxt<'_, 'tcx>, ty: Ty<'tcx>) -> LTy<'tcx> {
-    acx.lcx().label(ty, &mut |ty| match ty.kind() {
-        TyKind::Ref(_, _, _) | TyKind::RawPtr(_) => acx.new_pointer(),
-        _ => PointerId::NONE,
-    })
+trait AssignPointerIds<'tcx> {
+    fn lcx(&self) -> LTyCtxt<'tcx>;
+    fn new_pointer(&mut self) -> PointerId;
```

Comment:
```suggestion
    fn lcx(&self) -> LTyCtxt<'tcx>;
    
    fn new_pointer(&mut self) -> PointerId;
```

Reply:
Why was this resolved?  There's spacing between the other methods in this trait.

Reply:
Oh wait, I think I was looking at an outdated version.